console/frontend/src/main/frontend/src/app/views/monitors/monitors.service.ts

```diff
@@ -12,7 +12,7 @@ export type Monitor = {
   lastHit: string;
   triggers: Trigger[];
   displayName: string;
-  activeDestinations: boolean[];
+  activeDestinations: { [key: string]: boolean };
```

Comment:
Normally we use a util type called Record which means exactly the same as this one but is a bit stricter during type checking
```suggestion
  activeDestinations: Record<string, boolean>;
```